src/impl_methods.rs

```diff
@@ -1126,18 +1193,36 @@ impl<A, S, D> ArrayBase<S, D> where S: Data<Elem=A>, D: Dimension
     /// Return a mutable pointer to the first element in the array.
     #[inline(always)]
     pub fn as_mut_ptr(&mut self) -> *mut A
-        where S: DataMut
+        where S: DataRawMut
     {
-        self.ensure_unique(); // for RcArray
+        self.try_ensure_unique(); // for RcArray
         self.ptr
     }
 
+    /// Return a raw view of the array.
+    #[inline]
+    pub fn as_raw_view(&self) -> RawArrayView<A, D> {
+        unsafe { RawArrayView::new_(self.ptr, self.dim.clone(), self.strides.clone()) }
+    }
```

Comment:
The plain view method doesn't use `as_` in the name, it's just `view`. Can we do the same here? I guess as belongs to a convention but I don't see the benefit of it.

Reply:
Sure, that sounds good to me. Done.